### ISSUE TITLE
Image cache delete option

### ISSUE THREAD

**Issue by ZSF88** [gnosygnu/xowa#654]
Hi,

Not sure if this is the right place but I want to ask a few questions.
1. How do I delete image cache automatically downloaded by XOWA while my PC was connected to intenet? (I have image files but I disable them just to browse XOWA without images. I tried deleting image manually but they seems to reappear.)

2. Can I migrate database from old version of XOWA to new version? (I want to save ~200GB bandwidth this month.)

Mod please keep this post until above questions are answers. Thanks.

**Comment by gnosygnu** [comment]
Hi! Thanks for the interest in XOWA!

> How do I delete image cache automatically downloaded by XOWA while my PC was connected to intenet? (I have image files but I disable them just to browse XOWA without images. I tried deleting image manually but they seems to reappear.)

I think you want to look at the options on these 2 pages:
* home/wiki/Special:XowaCfg?grp=xowa.app.security
  * Web Access Enabled -> uncheck: This will stop downloading images while you're connecting to the internet
* home/wiki/Special:XowaCfg?grp=xowa.files.general
  * Maximum disk space used -> 5 MB : This will limit XOWA images to 5 MB. You can make this lower (0 MB?), but I haven't tested it
  * Clear Cache -> Click: This will force XOWA to manually delete all images until it's under the Max disk space used

> Can I migrate database from old version of XOWA to new version? (I want to save ~200GB bandwidth this month.)

Sorry, don't understand this question. I'm assuming you're talking about English Wikipedia. What dates are the old version (for example, 2018-07?) and the new version (for example, 2019-03?)?

**Comment by ZSF88** [comment]
Windows 10 threw me BSOD a few times so I couldn't get back here for a few days. Sorry about that.

Issue 1:
As you suggested, I set the minimum and maximum (disk space use) were set to 0:0, 0:1 and 1:1 respectively, cleared cache, refreshed, deleted the "file" folder, xowa.user.anonymous.sqlite3 file  and restarted the XOWA. However, those tiny cache files seems to persist. I tried deleting user folder as well. I discovered Web Access Enabled after images were download. That option was unchecked after that. 

Cached images are gone as soon as "Reduce cache to minimum" and "Clear cache" are applied but reappear after restarting XOWA. By the way, XOWA displays an error after those settings are applied. Video here: https://gofile.io/?c=8FFoji or https://file.io/NH40HZ (Lags because of display driver crash.)

Issue 2:
I have 158GB of XOWA files in folders en.wikinews.org, en.wikipedia.org, ....., species.wikipedia.org, www.wikidata.org, etc of "XOWA/wiki/" folder. Those are wiki from 2017.01 to 2017.12. Image here: https://i.imgur.com/56icchW.jpg . And want to move those files from XOWA v4.5.19 to new XOWA version. (so that the image cache issue can be resolved.) Is it possible?
TLDR: Want to move .xowa fiels from old XOWA to new XOWA. Is it possible?
